location/src/main/java/org/motechproject/nms/location/domain/HealthBlock.java

```diff
@@ -19,18 +19,18 @@
 public class HealthBlock extends MdsEntity {
 
     @Field
-    @UIDisplayable(position = 0)
+    @Column(allowsNull = "false")
```

Comment:
@Column(allowsNull = "false") adds a not null constraint to the database.  @NotNull does a validation in code and throws an exception before saving if the value is null.  So one is in code and the other is in the database.  (If I had to choose only one I would go with @Column